docs/source/step-by-step.rst

```diff
@@ -1,15 +1,760 @@
+a baseline performance reading.
+
+In your project folder (``proj/my_first_cfu``) run the following:
+
+.. code-block:: bash
+
+    $ make prog  # if you're not using the arty add TARGET=<your board>
+    $ make load  # if you're not using the arty add TARGET=<your board>
+
+After the build processes are completed you should see something like the
+following in your terminal:
+
+::
+
+    CFU Playground
+    ==============
+    1: TfLM Models menu
+    2: Functional CFU Tests
+    3: Project menu
+    4: Performance Counter Tests
+    5: TFLite Unit Tests
+    6: Benchmarks
+    7: Util Tests
+
+Navigate to the your model's menu by entering ``1``, then ``1`` in the
+menus. Once at the model menu (shown below) press ``g`` to run the golden
+tests.
+
+::
+
+    Tests for pdti8 model
+    =====================
+    1: Run with zeros input
+    2: Run with no-person input
+    3: Run with person input
+    g: Run golden tests (check for expected outputs)
+    x: eXit to previous menu
+    pdti8> g
+
+After running the golden tests you should have some output that looks like
+this:
+
+::
+
+    0,DEPTHWISE_CONV_2D,8074244
+    1,DEPTHWISE_CONV_2D,8261757
+    2,CONV_2D,11953686
+    3,DEPTHWISE_CONV_2D,4215107
+    4,CONV_2D,8476340
+    5,DEPTHWISE_CONV_2D,8176370
+    6,CONV_2D,13542375
+    7,DEPTHWISE_CONV_2D,2088669
+    8,CONV_2D,6907128
+    9,DEPTHWISE_CONV_2D,4107783
+    10,CONV_2D,11811746
+    11,DEPTHWISE_CONV_2D,1038518
+    12,CONV_2D,6088804
+    13,DEPTHWISE_CONV_2D,2003871
+    14,CONV_2D,11876651
+    15,DEPTHWISE_CONV_2D,1965732
+    16,CONV_2D,11839822
+    17,DEPTHWISE_CONV_2D,1950569
+    18,CONV_2D,11905968
+    19,DEPTHWISE_CONV_2D,1956535
+    20,CONV_2D,11883747
+    21,DEPTHWISE_CONV_2D,1989537
+    22,CONV_2D,11876834
+    23,DEPTHWISE_CONV_2D,515060
+    24,CONV_2D,6001072
+    25,DEPTHWISE_CONV_2D,941504
+    26,CONV_2D,11530926
+    27,AVERAGE_POOL_2D,55078
+    28,CONV_2D,13185
+    29,RESHAPE,1147
+    30,SOFTMAX,10300
+    Counter |  Total | Starts | Average |     Raw
+    ---------+--------+--------+---------+--------------
+        0    |     0  |     0  |   n/a   |            0
+        1    |     0  |     0  |   n/a   |            0
+        2    |     0  |     0  |   n/a   |            0
+        3    |     0  |     0  |   n/a   |            0
+        4    |     0  |     0  |   n/a   |            0
+        5    |     0  |     0  |   n/a   |            0
+        6    |     0  |     0  |   n/a   |            0
+        7    |     0  |     0  |   n/a   |            0
+       183M (   183087673) cycles total
+
+The comma-separated lines at the top signify the TensorFlow operation and the
+number of cycles it took to complete. The table at the bottom shows statistics
+from the performance CSRs (if you turned them on), the final line shows the
+total number of cycles spent during inference.
+
+Summing up all the cycle counts for the tensorFlow operations we see that
```

Comment:
The comma-separated-values list are intended to be copy-and-paste into spreadsheets. It would be worthwhile mentioning this - will certainly make summing and calcualting percentages simpler.